src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -98,10 +125,12 @@ class EndpointConnectionStats(
                 val entry = SsrcAndTimestamp(packet.senderSsrc, packet.senderInfo.compactedNtpTimestamp)
                 srSentTimes[entry] = clock.instant()
             }
+            // Sent TCC feedback reports loss on packets we *received*
+            is RtcpFbTccPacket -> processTcc(packet, receiveLossTracker)
```

Comment:
I intend to test the performance impact of this, because I think it might re-parse the packet. @JonathanLennox any thoughts? Has parsing TCC been a hotspot at all in any of your tests?

Reply:
But when we create a TCC packet, we use the builder and then write all the data to the buf. That packet has not had its fields "parsed", so accessing them here would trigger the lazy, I think.